presto-spi/src/main/java/com/facebook/presto/spi/block/ArrayBlock.java

```diff
@@ -49,7 +49,7 @@ public static Block fromElementBlock(int positionCount, boolean[] valueIsNull, i
             if (length < 0) {
                 throw new IllegalArgumentException(format("Offset is not monotonically ascending. offsets[%s]=%s, offsets[%s]=%s", i, arrayOffset[i], i + 1, arrayOffset[i + 1]));
             }
-            if (valueIsNull[i] && length != 0) {
+            if (valueIsNull != null && valueIsNull[i] && length != 0) {
```

Comment:
would you mark the `valueIsNull` field and ctor parameter as `@Nullable`? (same in other classes)

Reply:
Marked the getter methods instead of the fields to match the convention in code base

Reply:
oh, sure, getters should be marked as well. Please mark the fields too. (this is common in the code base and helps understand the code)